src/hyperion/device_setup_plans/dcm_pitch_roll_mirror_adjuster.py

```diff
@@ -120,9 +122,10 @@ def adjust_dcm_pitch_roll_vfm_from_lut(
     yield from bps.wait(DCM_GROUP)
 
     # VFM Adjust - for I03 this table always returns the same value
+    assert (vfm_lut := vfm.bragg_to_lat_lookup_table_path) is not None
```

Comment:
I think it's bad to use the walrus operator inside asserts. If you run the code with -O, asserts are disabled and then the behaviour of the application changes.

Reply:
I agree, I will change it here but we do have other places that do the same in the codebase, I've added https://github.com/DiamondLightSource/hyperion/issues/1342